.github/workflows/Dockerfile.dev.yml

```diff
@@ -153,14 +172,15 @@ jobs:
         go run test_horizon_ingesting.go
         curl http://localhost:8000
     - name: Run friendbot test
-      # Horizon with captive-core is currently broken. Don't run the Friendbot
-      # test in captive-core mode since it will always fail while Horizon is not
-      # functioning with captive-core.
-      # See https://github.com/stellar/quickstart/issues/328 for more details.
-      if: ${{ matrix.network == 'standalone' && !contains(matrix.options, '--enable-horizon-captive-core') }}
```

Comment:
Is this fixed now? Nice!

Reply:
that issue was closed with more details on needing to wait longer for horizon to get first checkpoint archive, i added `startretries=100` to friendbot supervisor conf and during testing the container observed friendbot service retries initially and after first archive file gets created, can see horizon ingestion clicked on, and then friendbot switched over to running state.

Reply:
You could make this change in a separate branch and get it merged to master to keep it out of this PR then, since it seems mostly unrelated to Soroban.